### PR TITLE
fix(progress-button): fix styling & add playground page

### PR DESCRIPTION
**Pull request by sergiubologa**
Before:
image

After:
image

### CODE REVIEW

File: projects/angular/directives/ui-progress-button/src/internal/ui-button-progress-bar.component.scss [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
     .mat-mdc-progress-bar {
         position: absolute;
         top: 0;
         left: 0;
     }
 }
-
-.ui-button-progress {
-    overflow: hidden;
-}
```
Comment (sergiubologa) [inline projects/angular/directives/ui-progress-button/src/internal/ui-button-progress-bar.component.scss:14]: this doesn't seem to be needed. If anybody knows why it was there please let me know.

**Review by anatolie-darii — APPROVED** [review]

**Review by ioanajoj — APPROVED** [review]

File: projects/angular/directives/ui-progress-button/src/internal/ui-button-progress-bar.component.scss [hunk 2]
```diff
@@ -1,14 +1,12 @@
 ui-button-progress-bar {
     display: block;
-    width: 100%;
+    width: 0;
```
Comment (LiviuTrifoi) [inline projects/angular/directives/ui-progress-button/src/internal/ui-button-progress-bar.component.scss:3]: why is width explicitly set to 0?

Comment (sergiubologa) [reply projects/angular/directives/ui-progress-button/src/internal/ui-button-progress-bar.component.scss:3]: so it doesn't occupy space inside the button

File: projects/angular/directives/ui-progress-button/src/internal/ui-button-progress-bar.component.scss (hunk 1 above)
Comment (anbalase) [reply projects/angular/directives/ui-progress-button/src/internal/ui-button-progress-bar.component.scss:14]: is the progress cropped? (corners are rounded?)

Comment (anbalase) [reply projects/angular/directives/ui-progress-button/src/internal/ui-button-progress-bar.component.scss:14]: maybe that was the point

Comment (sergiubologa) [reply projects/angular/directives/ui-progress-button/src/internal/ui-button-progress-bar.component.scss:14]: nope, it looks good

Comment (sergiubologa) [reply projects/angular/directives/ui-progress-button/src/internal/ui-button-progress-bar.component.scss]: actually, it was needed to not hide the button's rounded corners. I've added it back now.

**Review by anbalase — APPROVED** [review]

**Pull request merged**